documents/plan-lism-skill-update.md

```
@@ -70,14 +76,14 @@ skills/
 
 guide は「読む資料」ではなく、新規実装の**手順制御**も担う。SKILL.md に以下を置く。
 
-**(A) 実装フロー（厳守・forward）** — refactor の 0〜7（§5-2）と対称：
+**(A) 実装フロー（厳守・forward）** — refactor の順序パス（§5-3 Pass 2〜9）を**前向きに写像**した authoring workflow（手順数は guide=0〜4／refactor=0〜7 で対称ではない。対応するのは各 Pass ↔ FP）：
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)